queries/strace/highlights.scm

```diff
@@ -0,0 +1,54 @@
+[
+  "killed"
+  "by"
+  "exited"
+  "with"
+  "<unfinished ...>"
+  "<..."
+  "resumed>"
+] @keyword
+
+[
+  (errorName)
+  (errorDescription)
+] @exception
+
+(syscall) @function.builtin
+
+; Literals
+
+(integer) @number
+
+(pointer) @number.special
```

Comment:
`@constant` ?

Reply:
it's a hex number representing a pointer, idrt constant fits